src/lesson_12/hometask_12/Student.java

```diff
@@ -0,0 +1,70 @@
+package lesson_12.hometask_12;
+
+import java.io.File;
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Student {
+    private String name;
+    ArrayList<File> fileList = new ArrayList<>();
+    private int score;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getScore() {
+        return score;
+    }
+
+    public void setScore(int score) {
+        this.score = score;
+    }
+
+    public Student(String name, ArrayList<String> fileList, int score) {
```

Comment:
For students you need deir names. the files must be in array and must be max 5.
the score must be set to 0 at the beggining.